base-eslintrc.json

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
how would you feel about putting this in the configs directory since it's not getting picked up by default anyway?

Reply:
I like the idea. I will make the change